src/StatusQ/Controls/StatusBaseButton.qml

```diff
@@ -1,10 +1,18 @@
 import QtQuick 2.14
+import QtQuick.Controls 2.12
```

Comment:
The same as before. Qt creator sometimes dont understand which modules is default.

Reply:
please see above, you mean QtQuick or QtQuick.Controls import?